lib/matplotlib/tests/test_axes.py

```diff
@@ -7811,6 +7811,36 @@ def test_ytickcolor_is_not_yticklabelcolor():
         assert tick.label1.get_color() == 'blue'
 
 
+def test_xticklabelcolor_if_not_xtickcolor():
+    plt.rcParams['xtick.labelcolor'] = 'blue'
+    ax = plt.axes()
+    ticks = ax.xaxis.get_major_ticks()
+    for tick in ticks:
+        assert tick.label1.get_color() == 'blue'
```

Comment:
This part seems redundant with the above test.

Reply:
Thanks for pointing that out! I have updated the tests.